### PR TITLE
Clean up Rakefile

### PR DESCRIPTION
**Pull request by raccube**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
-require File.expand_path('../config/application', __FILE__)
+require File.expand_path("config/application", __dir__)
 
 Rails.application.load_tasks
 
 namespace :justask do
   desc "Upload to AWS"
   task paperclaws: :environment do
```
Comment (nilsding) [inline Rakefile]: yeet this

File: Rakefile [hunk 2]
```diff
@@ -39,211 +39,216 @@ namespace :justask do
 
   desc "Recount everything!"
   task recount: :environment do
```
Comment (nilsding) [inline Rakefile]: yeet this

File: Rakefile [hunk 3]
```diff
@@ -39,211 +39,216 @@ namespace :justask do
...
 
-    user = User.find_by_screen_name(args[:screen_name])
+    user = User.find_by(screen_name: args[:screen_name])
     abort "user #{args[:screen_name]} not found" if user.nil?
 
     user.remove_role :moderator
     puts "#{user.screen_name} is no longer a moderator."
   end
 
   desc "Hits an user with the banhammer."
```
Comment (nilsding) [inline Rakefile]: yeet this

File: Rakefile [hunk 4]
```diff
@@ -39,211 +39,216 @@ namespace :justask do
...
       target_user_id: user.id,
-      expiry: nil,
-      reason: args[:reason],
+      expiry:         nil,
+      reason:         args[:reason]
     )
     puts "#{user.screen_name} got hit by\033[5m YE OLDE BANHAMMER\033[0m!!1!"
   end
 
   desc "Hits an user with the banhammer for one day."
```
Comment (nilsding) [inline Rakefile]: yeet this

File: Rakefile [hunk 5]
```diff
@@ -39,211 +39,216 @@ namespace :justask do
...
       target_user_id: user.id,
-      expiry: DateTime.current + 1,
-      reason: args[:reason],
+      expiry:         DateTime.current + 1,
+      reason:         args[:reason]
     )
     puts "#{user.screen_name} got hit by\033[5m YE OLDE BANHAMMER\033[0m!!1!"
   end
 
   desc "Hits an user with the banhammer for one week."
```
Comment (nilsding) [inline Rakefile]: yeet this

File: Rakefile [hunk 6]
```diff
@@ -39,211 +39,216 @@ namespace :justask do
...
-      expiry: DateTime.current + 7,
-      reason: args[:reason],
-      )
+      expiry:         DateTime.current + 7,
+      reason:         args[:reason]
+    )
     puts "#{user.screen_name} got hit by\033[5m YE OLDE BANHAMMER\033[0m!!1!"
   end
 
   desc "Hits an user with the banhammer for one month."
```
Comment (nilsding) [inline Rakefile]: yeet this

File: Rakefile [hunk 7]
```diff
@@ -39,211 +39,216 @@ namespace :justask do
...
-      expiry: DateTime.current + 30,
-      reason: args[:reason],
-      )
+      expiry:         DateTime.current + 30,
+      reason:         args[:reason]
+    )
     puts "#{user.screen_name} got hit by\033[5m YE OLDE BANHAMMER\033[0m!!1!"
   end
 
   desc "Hits an user with the banhammer for one year."
```
Comment (nilsding) [inline Rakefile]: yeet this

File: Rakefile [hunk 8]
```diff
@@ -39,211 +39,216 @@ namespace :justask do
...
       target_user_id: user.id,
-      expiry: DateTime.current + 365_000_000_000,
-      reason: args[:reason],
+      expiry:         DateTime.current + 365_000_000_000,
+      reason:         args[:reason]
     )
     puts "#{user.screen_name} got hit by\033[5m YE OLDE BANHAMMER\033[0m!!1!"
   end
 
   desc "Removes banned status from an user."
```
Comment (nilsding) [inline Rakefile]: yeet this

File: Rakefile [hunk 9]
```diff
@@ -39,211 +39,216 @@ namespace :justask do
...
       target_user_id: user.id,
-      expiry: DateTime.current + 365,
-      reason: args[:reason],
+      expiry:         DateTime.current + 365,
+      reason:         args[:reason]
     )
     puts "#{user.screen_name} got hit by\033[5m YE OLDE BANHAMMER\033[0m!!1!"
   end
 
   desc "Hits an user with the banhammer for one aeon."
```
Comment (nilsding) [inline Rakefile]: yeet this

File: Rakefile [hunk 10]
```diff
@@ -39,211 +39,216 @@ namespace :justask do
...
     )
     puts "#{user.screen_name} got hit by\033[5m YE OLDE BANHAMMER\033[0m!!1!"
   end
 
   desc "Removes banned status from an user."
-  task :unban, [:screen_name] => :environment do |t, args|
-    fail "screen name required" if args[:screen_name].nil?
-    user = User.find_by_screen_name(args[:screen_name])
-    fail "user #{args[:screen_name]} not found" if user.nil?
+  task :unban, [:screen_name] => :environment do |_t, args|
```
Comment (nilsding) [inline Rakefile]: yeet this

File: Rakefile [hunk 11]
```diff
@@ -39,211 +39,216 @@ namespace :justask do
...
+    raise "screen name required" if args[:screen_name].nil?
+
+    user = User.find_by(screen_name: args[:screen_name])
+    raise "user #{args[:screen_name]} not found" if user.nil?
+
     UseCase::User::Unban.call(user.id)
     puts "#{user.screen_name} is no longer banned."
   end
 
   desc "Lists all users."
```
Comment (nilsding) [inline Rakefile]: absolutely yeet this

File: Rakefile [hunk 12]
```diff
@@ -39,211 +39,216 @@ namespace :justask do
...
     removed = User.where(confirmed_at: nil)
-        .where("confirmation_sent_at < ?", DateTime.now.utc - 3.months)
-        .destroy_all.count
+                  .where("confirmation_sent_at < ?", DateTime.now.utc - 3.months)
+                  .destroy_all.count
     puts "Removed #{removed} users"
   end
 
   desc "Prints lonely people."
   task loners: :environment do
```
Comment (nilsding) [inline Rakefile]: yeet this

**Review by nilsding — CHANGES_REQUESTED** [review]
# yeet

**Review by nilsding — APPROVED** [review]

**Review by pixeldesu — APPROVED** [review]

**Review by nilsding — COMMENTED** [review]
böse

**Pull request merged**